index.html

```diff
@@ -207,7 +212,7 @@ <h2 class="headings">Tonics</h2>
 
 <div class="dsk-2">  
  
-<h5 class="headings">Tonics</h5>
+<h5 class="headings">Uber Navigation</h5>
   
```

Comment:
<img width="1272" alt="2" src="https://github.com/hetrox8/portfolio1/assets/77843050/e1d850de-d864-45c9-84ac-0682c1d86c23">

- [ ] As mentioned by the previous reviewer, please make sure your cards are well aligned as in the figma design, this specific card is behaving a bit more abnormally than others, you should push it a bit to the left.

<img width="1230" alt="3" src="https://github.com/hetrox8/portfolio1/assets/77843050/49b6e50a-0643-42e4-ada6-2ce137368b38">

- [ ] As mentioned by the previous reviewer, please take a look back at the figma design to align some of the components as this one here, follow the arrow direction, 

<img width="445" alt="4" src="https://github.com/hetrox8/portfolio1/assets/77843050/4ac02feb-a45b-47f0-8d79-e6faf7a7463e">

- [ ] Same as here, these cards are a bit shorter in your design compared to the figma ones, please observe closely the original template to update this

<img width="588" alt="5" src="https://github.com/hetrox8/portfolio1/assets/77843050/dc633b57-1738-453f-b3af-ea0f3a17bc55">

- [ ] In the desktop view, the button form is aligned in the middle not on the left as you did, please update it as said the previous reviewer